src/SpellsReference/Controllers/SpellbookController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using SpellsReference.Data;
+using SpellsReference.Data.Repositories;
+using SpellsReference.Models;
+using System.Collections.Generic;
+using System.Web.Mvc;
+
+namespace SpellsReference.Controllers
+{
+    [Authorize]
+    public class SpellbookController : Controller
+    {
+        private IContext context;
+
+        public SpellbookController(IContext context)
+        {
+            this.context = context;
+        }
```

Comment:
Again, we should inject the repository dependency here instead of instantiating new repository objects.